demo/src/docs/Colors.js

```diff
@@ -0,0 +1,224 @@
+// ignore
+import React from "react";
+import Typography from "elevate-ui/Typography";
+import withStyles from "elevate-ui/withStyles";
+import { PrismCode } from "react-prism";
+import CheckCircle from "elevate-ui-icons/CheckCircle";
+
+import { Link } from "react-router-dom";
+
+const Colors = ({ classes }) => (
+  <div className={classes.wrapper}>
+    <div className={classes.headingContainer}>
+      <Typography type="title" gutterBottom>{`Colors`}</Typography>
+      <Typography type="body" className={classes.subheading}>
+        This is how you would reference Elevate's Primary Red for a heading:
+      </Typography>
+    </div>
+    <div>
+      <div>
+        <PrismCode component="pre" className="language-javascript">
+          {`
+  import React from "react";
+  import withStyles from "elevate-ui/withStyles";
+  import Typography from "elevate-ui/Typography";
+  
+  const Heading = ({ classes }) => (
+    <div>
+      <Typography type="heading4" className={classes.heading}>
+        Hello there 👋
+      </Typography>
+    </div>
+  );
+  
+  export default withStyles((theme) => ({
```

Comment:
Just a thought,

I know this is how we export the components in EUI, but in our actual applications we usually do:

```js
const styles = (theme) => ({
  root: {
    color: theme.colors.primary['400'],
  }
});

export default withStyles(styles)(SomeComponent);
```

Maybe it would be worth updating the doc markup to match this pattern? It also makes it a bit easier to understand.